index.html

```diff
@@ -1,27 +1,84 @@
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js"></script>
     <title>Document</title>
 </head>
-<body class="container">
-    <header class="w-100 d-flex justify-content-center">
-        <h1>Awesome books</h1>
+
+<body>
+    <header class="mb-5 d-flex flex-column">
+        <nav class="navbar navbar-expand-lg navbar-light bg-light border">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#">Awesome Books</a>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav  ms-auto mb-2 mb-lg-0">
+                        <li class="nav-item border-start text-center">
+                            <a id="listLink" class="nav-link active" href="#">List</a>
+                        </li>
+                        <li class="nav-item border-start text-center">
+                            <a id="addLink" class="nav-link" href="#">Add New</a>
+                        </li>
+                        <li class="nav-item border-start text-center">
+                            <a id="contactLink" class="nav-link" href="#">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        <span id="date" class="align-self-end p-2"></span>
     </header>
-        <section class="book border border-4">
-        </section>
 
-        <section class="w-100 d-flex justify-content-center my-5">
+    <section id="date-time" class="d-flex justify-content-end mb-5 mx-5 mt-0" ></section>
+
+    <section class="container">
+        <section id="show" class="book border border-4">
+            <section class="w-100 d-flex justify-content-center">
+                <h1>Awesome books</h1>
+            </section>    
+        </section>
+        
+        <section id="add" class="d-none w-100 d-flex justify-content-center my-5">
             <form id="form-add" class="w-50 d-flex flex-column justify-content-center">
                 <h2 class="text-center">Add a new book</h2>
                 <input type="text" class="my-2 form-control" placeholder="Title" id="title" name="title">
                 <input type="text" class="my-2 form-control" placeholder="Author" id="author" name="author">
                 <button type="submit" class="w-25 align-self-end my-2 btn btn-success" id="add-btn">Add</button>
             </form>
-        </section>  
+        </section>
```

Comment:
- [ ] Great job for sure, let us add `required` in our input field to prevent the user to enter empty data, after completing this you will be able to move to the next milestone.

![calcc5](https://user-images.githubusercontent.com/55635977/133688123-c2bea203-ff9d-4100-a888-64f7a896d74d.png)

Reply:
This issue is well resolved! ✔️